### PR TITLE
[fmt] Format topology module and tests

### PR DESCRIPTION
**Pull request by RMeli**
Format topology module and tests. Some files from active PRs might be excluded (see #4815).

----
📚 Documentation preview 📚: https://mdanalysis--4849.org.readthedocs.build/en/4849/

### CODE REVIEW

File: package/MDAnalysis/topology/tpr/setting.py
Comment (RMeli) [inline package/MDAnalysis/topology/tpr/setting.py]: Does formatting here has a special meaning?

File: package/MDAnalysis/topology/tpr/setting.py
Comment (RMeli) [inline package/MDAnalysis/topology/tpr/setting.py:107]: Does formatting here has a special meaning?

File: testsuite/MDAnalysisTests/topology/test_pdb.py
Comment (RMeli) [inline testsuite/MDAnalysisTests/topology/test_pdb.py]: Not great, not terrible.

File: testsuite/MDAnalysisTests/topology/test_pdb.py
Comment (RMeli) [inline testsuite/MDAnalysisTests/topology/test_pdb.py]: Not great, not terrible.

File: testsuite/MDAnalysisTests/topology/test_top.py
Comment (RMeli) [inline testsuite/MDAnalysisTests/topology/test_top.py]: Not great, not terrible.

File: testsuite/MDAnalysisTests/topology/test_tprparser.py
Comment (RMeli) [inline testsuite/MDAnalysisTests/topology/test_tprparser.py]: OK?

File: testsuite/MDAnalysisTests/topology/test_tprparser.py
Comment (RMeli) [inline testsuite/MDAnalysisTests/topology/test_tprparser.py]: Use the trick above?

File: testsuite/MDAnalysisTests/topology/test_tprparser.py
Comment (RMeli) [inline testsuite/MDAnalysisTests/topology/test_tprparser.py]: I don't mind, but not everyone will like this.

File: testsuite/MDAnalysisTests/topology/test_tprparser.py
Comment (RMeli) [inline testsuite/MDAnalysisTests/topology/test_tprparser.py:377]: This is a nice trick, could be applied elsewhere. Here just `list("HCHHC")` should work since they are all single-letter elements, but this will work elsewhere.

File: testsuite/MDAnalysisTests/topology/test_tprparser.py
Comment (RMeli) [inline testsuite/MDAnalysisTests/topology/test_tprparser.py]: Is there some meaning here?

File: testsuite/MDAnalysisTests/topology/test_top.py
Comment (RMeli) [inline testsuite/MDAnalysisTests/topology/test_top.py]: If someone judge in terms of LOC, this is a win. =P

**Review by RMeli — COMMENTED** [review]
The web UI was really slow with this one. I hope all the review comments were saved.

**Review by orbeckst — APPROVED** [review]
blanket LGTM — please merge when you're happy with it @RMeli 

🎄 🎁

File: package/MDAnalysis/topology/tpr/setting.py
Comment (RMeli) [reply package/MDAnalysis/topology/tpr/setting.py]: Retained formatting.

File: package/MDAnalysis/topology/tpr/setting.py
Comment (RMeli) [reply package/MDAnalysis/topology/tpr/setting.py:107]: Restored formatted.

File: testsuite/MDAnalysisTests/topology/test_pdb.py
Comment (RMeli) [reply testsuite/MDAnalysisTests/topology/test_pdb.py]: Turned formatted off.

File: testsuite/MDAnalysisTests/topology/test_pdb.py
Comment (RMeli) [reply testsuite/MDAnalysisTests/topology/test_pdb.py]: Turned formatted off.

File: testsuite/MDAnalysisTests/topology/test_top.py
Comment (RMeli) [reply testsuite/MDAnalysisTests/topology/test_top.py]: Actually, quite terrible. Formatted manually.

**Comment by RMeli** [comment]
Wait for https://github.com/MDAnalysis/mdanalysis/pull/4866 before merging this one.

File: testsuite/MDAnalysisTests/topology/test_top.py
Comment (RMeli) [reply testsuite/MDAnalysisTests/topology/test_top.py]: Formatted manually.

File: testsuite/MDAnalysisTests/topology/test_tprparser.py
Comment (RMeli) [reply testsuite/MDAnalysisTests/topology/test_tprparser.py]: Formatted manually.

File: testsuite/MDAnalysisTests/topology/test_tprparser.py
Comment (RMeli) [reply testsuite/MDAnalysisTests/topology/test_tprparser.py]: Left original formatting.

File: testsuite/MDAnalysisTests/topology/test_tprparser.py
Comment (RMeli) [reply testsuite/MDAnalysisTests/topology/test_tprparser.py]: Formatted manually.

File: testsuite/MDAnalysisTests/topology/test_tprparser.py
Comment (RMeli) [reply testsuite/MDAnalysisTests/topology/test_tprparser.py]: Formatted manually.

**Comment by RMeli** [comment]
@marinegor this should be good for final review.

**Comment by RMeli** [comment]
Wait for https://github.com/MDAnalysis/mdanalysis/pull/4873 before merging.

**Review by marinegor — COMMENTED** [review]
retrospective LGTM

**Pull request merged**
